apps/web/src/pages/settings/SettingsPageContainer.tsx

```diff
@@ -24,10 +25,23 @@ const SettingsPageContentContainer: FC<PropsWithChildren<{}>> = ({ children }) =
   );
 };
 
-export const SettingsPageContainer: FC<PropsWithChildren<ISettingsPageContainerProps>> = ({ title, children }) => {
+export const SettingsPageContainer: FC<PropsWithChildren<ISettingsPageContainerProps>> = ({
+  title,
+  children,
+  header,
+}) => {
   return (
     <PageContainer title={title}>
       <PageHeader title={title} />
+      <When truthy={!!header}>
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** I think Nikolay may have established a new layout pattern on this page (and only this one), so this supports it without changing other settings pages‏